Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -2217,6 +2217,16 @@ def prepare(self, doChecking = False):
 
 	self.pythonCfgCode += self.addCustomiseCmdLine()
 
+        # Temporary hack to put the early delete customization after
+        # everything else
+        #
+        # FIXME: remove when no longer needed
+        if "RECO" in self.stepMap or "RAW2RECO" in self.stepMap:
```

Comment:
Hi @makortel a comment and question here

1) the same changes need to be applied to self.process

2) can't we just remove the if statement to future proof this?